velox/exec/fuzzer/CMakeLists.txt

```diff
@@ -27,6 +27,12 @@ add_library(
   PrestoSql.cpp
 )
 
+# Add VeloxQueryRunner only when remote functions are enabled
+if(VELOX_ENABLE_REMOTE_FUNCTIONS)
+  target_sources(velox_fuzzer_util PRIVATE VeloxQueryRunner.cpp)
+  target_link_libraries(velox_fuzzer_util FBThrift::thriftcpp2)
+endif()
+
```

Comment:
@peterenescu What's our plan to resolve this? Can we find out what're missing for building VeloxQueryRunner in Github CI and find an approach to bridge the gap? We would want to run fuzzer with VeloxQueryRunner in Github CI eventually too.